service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FeedbackReceiver.java

```diff
@@ -74,7 +72,7 @@
      * @param iotHubServiceClientProtocol protocol to be used
      * 
      */
-    public FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol)
+    public FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, ProxyOptions proxyOptions)
```

Comment:
Breaking change?

Reply:
Good catch, here and on the other sub-service client classes. I'll add an overload constructor that takes the proxy options instead